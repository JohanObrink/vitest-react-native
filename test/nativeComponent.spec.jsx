import { test } from 'vitest'
import { render } from '@testing-library/react-native'
import Item from './src/Item'

test('native components render correctly', () => {
  const container = render(<Item />)
  expect(container).toMatchSnapshot()
})