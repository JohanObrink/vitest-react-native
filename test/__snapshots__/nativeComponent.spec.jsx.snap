// Vitest Snapshot v1

exports[`native components render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#f7f8fa",
      "flexDirection": "row",
      "paddingBottom": 15,
      "paddingHorizontal": 15,
      "paddingTop": 10,
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    hitSlop={10}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      [
        {
          "opacity": 0.8,
        },
        false,
      ]
    }
  >
    </test/assets/trashIcon.svg
      height={20}
    />
  </View>
</View>
`;
